README.md

```diff
@@ -364,14 +364,15 @@ duckpan new MyFirst::Spice
 ---
 
 ```
-duckpan query [package_name ...]
+duckpan query [package_name ... | instant_inswer_id ...]
 ```
 
 Test Goodie and Spice triggers interactively on the command line.
 
 Arguments:
 
-- `[package_name ...]` to load only the specified Spice or Goodie Instant Answers.
+- `[package_name ...]` to load one or more Spice, or Goodie Instant Answers.
+- `[instant_answer_id ...]` to load one or more Spice, or Goodie Instant Answers; Or load a single Fathead Instant Answer
```

Comment:
Could we just say `name` or something? The forms supported are probably too many to name (ID, package, sub-package, name etc.) - but they are all 'names'. In any case, we should probably promote the ID over the other names (it is easier to 'standardize').